src/core/symbology-ng/qgssymbolv2.cpp

```diff
@@ -35,6 +37,33 @@
 
 #include <cmath>
 
+inline
+QString rotateEnMasse( double additionalRotation, const QString & exprStr )
```

Comment:
as per an earlier version of this PR, can these "enMasse" methods/classes all be renamed to avoid the use of "enMasse"? Something like "rotateWholeSymbol" or "rotateAllSymbolLayers" would be easier to understand and result in a nicer API